### PR TITLE
added note on single-node allocator warning in readme

### PR DESCRIPTION
**Pull request by lpambagoda**

### CODE REVIEW

**Comment by dimitriwalters** [comment]
+1

Perhaps indicating in the message that it's a warning too, though it very well could be a failure.

File: README.md [hunk 1]
```diff
@@ -85,6 +85,13 @@ DIR=$(mktemp -d /tmp/dbXXXXXX)
 ```
 This initializes and starts a single-node cluster in the background.
 
+By default, a range in CockroachDB wants to replicate to a minimum of three nodes. You can run a single node, but you will receive the following error:
+
+```bash
```
Comment (tamird) [inline README.md]: remove "bash"; this is not bash

**Comment by tamird** [comment]
LGTM modulo nit

File: README.md (hunk 1 above)
Comment (lpambagoda) [inline README.md]: done

**Comment by lpambagoda** [comment]
I think we can ignore this since Jess is already addressed this under  https://github.com/cockroachdb/cockroach/pull/2779

**Comment by jess-edwards** [comment]
Thanks!

**Pull request merged**